app/src/main/kotlin/com/isaac/pokedex_clone/presentation/favourite/FavouriteFragment.kt

```diff
@@ -79,6 +78,11 @@ class FavouriteFragment : BaseFragment<FragmentFavouriteBinding>(FragmentFavouri
         itemTouchHelper.attachToRecyclerView(binding.rvPokemon)
     }
 
+    override fun onResume() {
```

Comment:
ở OnResumed call lại getListFavourite này mỗi khi chuyển tab khả năng cái loadingManager hiển thị nhiều sẽ gây ức chế => em nghĩ là tìm phương án khác

Reply:
->  Này em nghĩ chắc để nó ở global viewmodel như HomeViewModel đi anh, để nó refect toàn bộ app luôn